core/modules/gtpu_encap.cc

```diff
@@ -201,8 +201,7 @@ CommandResponse GtpuEncap::Init(const bess::pb::GtpuEncapArg &arg) {
   tout_uport = AddMetadataAttr("tunnel_out_udp_port", sizeof(uint16_t),
                                AccessMode::kRead);
   DLOG(INFO) << "tout_uport: " << tout_uport << std::endl;
-  qfi_attr =
-      AddMetadataAttr("qfi", sizeof(uint8_t), AccessMode::kRead);
+  qfi_attr = AddMetadataAttr("qfi", sizeof(uint8_t), AccessMode::kRead);
   DLOG(INFO) << "qfi_attr: " << qfi_attr << std::endl;
```

Comment:
does'nt seem to have any impact logically?

Reply:
![image](https://user-images.githubusercontent.com/6157640/128481080-a398c57c-1a9a-4849-9bcf-470c874fe9ce.png)